frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/option_action.dart

```diff
@@ -30,6 +48,98 @@ class DividerOptionAction extends CustomActionCell {
   }
 }
 
+class AlignOptionAction extends PopoverActionCell {
+  AlignOptionAction({
+    required this.editorState,
+  });
+
+  final EditorState editorState;
+
+  @override
+  Widget? leftIcon(Color iconColor) {
+    return FlowySvg(
+      name: 'editor/align/$align',
+      size: const Size.square(12),
+    ).padding(all: 2.0);
+  }
+
+  @override
+  String get name {
+    return LocaleKeys.document_plugins_optionAction_align.tr();
+  }
+
+  @override
+  Widget Function(
+    BuildContext context,
+    PopoverController parentController,
+    PopoverController controller,
+  ) get builder => (context, parentController, controller) {
+        final selection = editorState.selection?.normalized;
+        if (selection == null) {
+          return const SizedBox.shrink();
+        }
+        final node = editorState.getNodeAtPath(selection.start.path);
+        if (node == null) {
+          return const SizedBox.shrink();
+        }
+        final List<Widget> children =
+            OptionAlignType.values.map((e) => OptionAlignWrapper(e)).map((e) {
+          final leftIcon = e.leftIcon(Theme.of(context).colorScheme.onSurface);
+          final rightIcon =
+              e.rightIcon(Theme.of(context).colorScheme.onSurface);
+          return HoverButton(
+            onTap: () async {
+              await onAlignChanged(e.inner);
+              controller.close();
+              parentController.close();
```

Comment:
Could we throw this in a private method?

Reply:
Is it necessary to put this into private function? Just two lines.